sound/soc/intel/boards/sof_da7219_max98373.c

```diff
@@ -146,17 +156,24 @@ static int da7219_codec_init(struct snd_soc_pcm_runtime *rtd)
 
 static int speaker_amp_init(struct snd_soc_pcm_runtime *rtd)
 {
-	int ret;
+	const struct snd_soc_dapm_widget *snd_widget = max98373_widgets;
+	const struct snd_soc_dapm_route *snd_route = max98373_map;
+	int ret, size = 2;
```

Comment:
does it make sense to says size = ARRAY_SIZE(max98373_widgets) here? and ....

Reply:
Ack.